release/common/base-chart.component.d.ts

```diff
@@ -13,6 +13,7 @@ export declare class BaseChartComponent implements OnChanges, AfterViewInit, OnD
     scheme: any;
     schemeType: string;
     customColors: any;
+    legendTitle: string;
```

Comment:
Not all charts have legends, so we shouldn't put this here.

Let's put it as a property of legendOptions: https://github.com/swimlane/ngx-charts/blob/master/src/bar-chart/bar-horizontal.component.ts#L175